### DISCUSSION TITLE
exclude package from upgrade command

### DISCUSSION THREAD

**Discussion by floh96** [microsoft/winget-cli#3723] [Q&A] [answered]
How can i exclude the discord package from the upgrade command so that it is not shown in the output of the command?

![image](https://github.com/microsoft/winget-cli/assets/49693964/38a61d04-6f06-44f0-8e35-33520bf9eb27)

Is the package pinned? 
winget pin list says there are no pins defined.

**Comment by Trenly** [comment, marked as answer]
You can use `winget pin add <id>` to pin a package

What you are experiencing is that some packages, like Discord, have a flag set that when there is no pin, they appear in that separate section because upgrading them doesn't always work properly and so they are automatically excluded when using `--all` but can still be upgraded using `winget upgrade <id>`
